starenchenkonv/first/task3.py

```diff
@@ -0,0 +1,11 @@
+num_1 = float(input("type number: "))
+num_2 = float(input("type number: "))
+counter = 1
+num_1_int = int(num_1)
+num_2_int = int(num_2)
+
+for index in range(num_1_int, num_2_int + 1):
+    if index > 0:
+        counter = counter * index
+
+print(counter)
```

Comment:
num_1 = float(input("1 number: "))
num_2 = float(input("2 number: "))
counter = 1
num_1_int = int(num_1)
num_2_int = int(num_2)

for number in range(num_1_int, num_2_int + 1):
    if number > 0:
        counter = counter * number

print(counter)